src/core/core.scale.js

```diff
@@ -1599,8 +1599,8 @@ export default class Scale extends Element {
   _layers() {
     const me = this;
     const opts = me.options;
-    const tz = opts.ticks && opts.ticks.z || 0;
-    const gz = opts.grid && opts.grid.z || 0;
+    const tz = opts.ticks && opts.ticks.z || -1;
```

Comment:
Should use valueOrDefault, because you can't define z as `0` now. Does the tick z need to be changed?

Reply:
The test wont fail if it isnt changed, but I thought since they where at the same z-index it would be logic to keep them at the same z-index